Update/_mina_001.txt

```diff
@@ -410,12 +399,8 @@ void main()
 		   NULL, "I can't remember how long I've been living just for the sake of obtaining that simple happiness.", Line_Normal);
 	ClearMessage();
 
-	DrawScene( "Mo1", 0 );
-
-	DisableWindow();
-	DrawBustshot( 7, "filter_Hanyu", 0, 0, 0, FALSE, 0, 0, 0, 0, 0, 0, 0, 12, 400, TRUE );
-	DrawBustshotWithFiltering( 0, "Ha_YowakiA1%", "mask00", 1, 0, 0, FALSE, 0, 0, 0, 0, 0, 9*2, 300, TRUE );
-	DisplayWindow();
+	DrawBustshot( 0, "sprite/transparent/ha1_yowaki_", 0, 0, 0, FALSE, 0, 0, 0, 0, 0, 0, 0, 5, 0, FALSE );
```

Comment:
This is a mistake. DrawBustshot is not LS compatible so it needs the full path to the sprite - including the 0/1/2 at the end which is missing here. Grelo usually used 0. The number is not meaningless though - it changes which variant will be shown when LS is disabled.

Reply:
gotcha, fixed it all